staging/src/k8s.io/kubectl/pkg/cmd/util/factory_client_access.go

```diff
@@ -60,7 +75,12 @@ func NewFactory(clientGetter genericclioptions.RESTClientGetter) Factory {
 }
 
 func (f *factoryImpl) ToRESTConfig() (*restclient.Config, error) {
-	return f.clientGetter.ToRESTConfig()
+	cfg, err := f.clientGetter.ToRESTConfig()
+	if err != nil {
+		return nil, err
+	}
+	cfg.AcceptContentTypes = kubectlAcceptHeader()
```

Comment:
we don't want to switch to requesting proto table access globally ... only in the get command (https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/cmd/get/get.go#L434-L438)

Reply:
Switched - that said it may not help given we're using unstructured...